lib/main.coffee

```diff
@@ -31,9 +26,19 @@ module.exports =
     @subscriptions.add atom.config.observe 'linter-phpmd.rulesets',
       (rulesets) =>
         @rulesets = rulesets
-    @subscriptions.add atom.config.observe 'linter-phpmd.projectRules',
-      (projectRules) =>
-        @projectRules = projectRules
+        if atom.config.get('linter-phpmd.projectRules') is true
```

Comment:
Sorry, must have missed that earlier comment. `atom.config.get()` returns the _value_ of the specified key, in this case since the key is boolean it could be set to `false` (the default value), in which case that would never be true. You could do something like `atom.config.get('linter-phpmd').hasOwnProperty('projectRules')` to see if the key exists.